packages/validator/src/cli-validator/utils/print-results.js

```diff
@@ -17,65 +17,66 @@ module.exports = function print(context, results) {
   }
 
   const colors = {
-    error: 'bgRed',
-    warning: 'bgYellow',
-    info: 'bgGrey',
-    hint: 'bgGreen',
+    error: 'red',
+    warning: 'yellow',
+    info: 'green',
+    hint: 'grey',
   };
 
   types.forEach(type => {
     if (!results[type].summary.total) {
       return; // continue
     }
 
-    let color = colors[type];
-    logger.info(chalk[color].bold(`${type}s\n`));
-    // convert 'color' from a background color to foreground color
-    color = color.slice(2).toLowerCase(); // i.e. 'bgRed' -> 'red'
+    const color = colors[type];
+    console.log('');
+    console.log(chalk[color](`${type}s`));
```

Comment:
Since the primary output is now logger-independent, we may now want to set the default level of the root logger to "warn" or "error" and let the user decide if they want more verbose logs. What do you think?